### PR TITLE
Do not show action Picking Terminal V2 -> Picking -> Process if there are no rows available for processing

### PR DESCRIPTION
**Pull request by TheBestPessimist**
https://github.com/metasfresh/metasfresh/issues/6997

### CODE REVIEW

File: backend/metasfresh-webui-api/src/main/java/de/metas/ui/web/pickingV2/productsToPick/process/ProductsToPick_4EyesReview_ProcessAll.java [hunk 1]
```diff
@@ -76,23 +76,31 @@ protected ProcessPreconditionsResolution checkPreconditionsApplicable()
 			return ProcessPreconditionsResolution.rejectWithInternalReason("only picker shall be allowed to process");
 		}
 
+		final List<ProductsToPickRow> rows = getRowsNotAlreadyProcessed();
```
Comment (metas-rc) [inline backend/metasfresh-webui-api/src/main/java/de/metas/ui/web/pickingV2/productsToPick/process/ProductsToPick_4EyesReview_ProcessAll.java:79]: I think you should check if there are **unpacked** rows as well, not only the ones that are not already processed. If the rows were picked but not yet packed, the "Process"  action is available and one can set the shipper transportation and believe the stuff really got in there, when in fact they didn't.

Comment (TheBestPessimist) [reply backend/metasfresh-webui-api/src/main/java/de/metas/ui/web/pickingV2/productsToPick/process/ProductsToPick_4EyesReview_ProcessAll.java:79]: If there are Picked rows but not Packed, the Process action does not appear.

image

File: backend/metasfresh-webui-api/src/main/java/de/metas/ui/web/pickingV2/productsToPick/process/ProductsToPick_4EyesReview_ProcessAll.java [hunk 2]
```diff
@@ -205,8 +213,8 @@ private void deliverAndInvoice(final List<PickingCandidate> pickingCandidates)
 				.collect(ImmutableList.toImmutableList());
 	}
 
-	private boolean partialDeliveryNotAllowed()
+	private boolean partialDeliveryAllowed()
```
Comment (teosarca) [inline backend/metasfresh-webui-api/src/main/java/de/metas/ui/web/pickingV2/productsToPick/process/ProductsToPick_4EyesReview_ProcessAll.java:216]: why not prefixing with `is`?

Comment (TheBestPessimist) [reply backend/metasfresh-webui-api/src/main/java/de/metas/ui/web/pickingV2/productsToPick/process/ProductsToPick_4EyesReview_ProcessAll.java]: done

**Pull request merged**